examples/rasa/README.md

```diff
@@ -71,7 +71,7 @@ Therefore you would add the following to rasa-values.yaml:
 ## Extra environment variables used in the Rasa X/Enterprise configuration
 extraEnv:
  - name: RASA_MODEL_SERVER
-   value: http://example.com/api/projects/default/models/tags/production
+   value: http://rasa-x-rasa-x:5002/api/projects/default/models/tags/production
```

Comment:
Here I'd leave it as it was. Rasa Model Server can point to any deployment, not necessarily within the same cluster, or namespace.

Reply:
Doesn't it make sense to change it considering this file is titled **Using Rasa X/Enterprise deployed in the same namespace as a configuration endpoint**? 🤔

Reply:
Ah, ok, I didn't see the title, then all good.